src/jest-playwright-preset/global.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-console */
+import {
+  setup as setupServer,
+  teardown as teardownServer,
+  ERROR_TIMEOUT,
+  ERROR_NO_COMMAND,
```

Comment:
That doesn't appear to still be exported in `jest-process-manager`, but if it's the same as the old `jest-dev-server`, the value [should be](https://github.com/argos-ci/jest-puppeteer/blob/d49a6a127cf636ef398609f565dff5f537c80532/packages/jest-dev-server/src/global.js#L35) "ERROR_NO_COMMAND". So it could be swapped out.

Same for "ERROR_TIMEOUT".